<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageFormatSetup</class>
 <widget class="QDialog" name="ImageFormatSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Active image formats</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="5">
    <widget class="QComboBox" name="comboBoxImageType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Select artwork class to adjust</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonOk">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Apply image format setup and close dialog</string>
     </property>
     <property name="text">
      <string>&amp;Ok</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/yes.png</normaloff>:/data/img/yes.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Cancel image format setup</string>
     </property>
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/no.png</normaloff>:/data/img/no.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonRestore">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Restore active image formats from currently stored setup</string>
     </property>
     <property name="text">
      <string>&amp;Restore</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/undo.png</normaloff>:/data/img/undo.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>442</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="labelImageType">
     <property name="text">
      <string>Artwork class</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="toolTip">
      <string>Drag &amp; drop format entries to adjust priorities of activated formats for the current artwork class</string>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Format</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Extensions</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Activation</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeWidget</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonRestore</tabstop>
  <tabstop>comboBoxImageType</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections/>
</ui>
